### ISSUE TITLE
avoid text-encoding: chunked for small table

### ISSUE THREAD

**Issue by tiptronic** [directus/directus#2061]
I have a somewhat large table (not really large, but 1000 rows and 50 columns). The text-content of each row is less than 400 bytes/row.

Now when I query the table I always get a chunked response (and Error 500) - even if I just query 1 row:.

- table1 contains 6 fields
- table2 contains 50/40/30/20 (I deleted fields in batches of 10 -> when there are only 10 fields left, it starts working)

http://directus.test/api/1.1/tables/table1/rows        -> works
http://directus.test/api/1.1/tables/table2/rows       -> chunked
http://directus.test/api/1.1/tables/table2/rows?limit=1   -> chunked

I have to reduce the number of fields to approx 10, to get the response send unchunked.

If I put much more date into one of the fields of table1 - it is still send unchunked (although the request to 

http://directus.test/api/1.1/tables/table1/rows

now contains a multiple of the data as the reqsponse from table2...

So I guess, it's not (only) a Apache problem...

Is there anything we can avoid this on directus side?

thanks
andy

## Version Info
  - Directus version and branch: (Or commit hash)  6.4.0
  - PHP version:    7.2.1
  - MySQL version:  5.6.38
  - Web server: (Ex. Apache, nginx or IIS?)   Apache 2.2
  - OS name and version: macOSX 10.11

**Comment by WellingGuzman** [comment]
Hey @tiptronic thanks for the report, do you have any logs in `api/logs`? can you share the schema you are using so we can replicate this issue?

Can you show us the chunked response? or a screenshot of the error 500?

**Comment by tiptronic** [comment]
Hi @WellingGuzman ,
thanks for your reply (and also for the hint to ```api/logs``` 
I finally found out what the problem is/was:
directus throws an Exception

```PDOException: SQLSTATE[42000]: Syntax error or access violation: 1064``` 

 if the table-structure contains 'umlaut' characters. (I know, it's not a recommended way to name it's columns, but I got the table from elsewhere).
You can easily reproduce the error, with a small table:
```mysql
CREATE TABLE `something` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(255) DEFAULT NULL,
  `Kündigungsdatum` varchar(255) DEFAULT NULL,
  `geöffnet` varchar(255) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=1109 DEFAULT CHARSET=utf8;
```
Plz note the umlaut character in field 3 'Kündigungsdatum'...
Removing those characters made everything work!!
So sorry for the spam, but maybe there's a way to avoid that exception...
If you need more info or help, just let me know.
andy

**Comment by WellingGuzman** [comment]
I see, this is a "bug" with `zend-db`, I will keep this open to remind us to update it on our fork or wait on `zend-db` to fix it, whichever comes first.

**Comment by rijkvanzanten** [comment]
@WellingGuzman Is there an open issue over at the `zend-db` repo? What is the state of this? If it's not something we're in control of, we should probably move this issue there.

**Comment by WellingGuzman** [comment]
Closed by https://github.com/WellingGuzman/zend-db/commit/ef55371343a85e5cd0937d0ab9d30da7e86f5ab3. 

This will be fixed in the next version (6.4.10)

**Comment by benhaynes** [comment]
Kamino closed and cloned this issue to [directus/directus-6-legacy](https://github.com/directus/directus-6-legacy/issues/31)
